nodes/validator/main.py

```diff
@@ -16,12 +16,13 @@ async def transaction_new(request):
             file_path = f"{os.environ['MEMPOOL']}/{filename}.json"
             with open(file_path, 'w') as file:
                 json.dump(txn, file)
-        return web.Response(status = 200)
-    except:
-        return web.Response(status = 500)
+            return web.Response(status = 200)
+        else:
+            return web.Response(status = 500)
```

Comment:
This might allow incorrect HTTP response codes due to the two validity checks failing. If a validity check fails, it's the request's issue. If the file write fails, it's a `500`.

Reply:
return web.Response(status=200)
        else:
                return web.Response(status=418)  # I'M A TEAPOT
    except:
        return web.Response(status=500)

If file read it is correct return 200 which is ok,
if  file read something is not working, which is related users data would be 418
If server is not work 500   right ?

Reply:
It's more to the effect of if there's something wrong with the data of the _request_, we send `4xx`. If the server encounters and internal fault, `5xx`.